spec/anchor.spec.js

```diff
@@ -329,6 +329,42 @@ describe('Anchor Button TestCase', function () {
             expect(link).not.toBeNull();
             expect(link.href).toBe(validUrl.toLowerCase());
         });
+        it('should change spaces to %20 for a valid url if linkValidation options is set to true', function () {
+            var editor = this.newMediumEditor('.editor', {
+                anchor: {
+                    linkValidation: true
```

Comment:
Good idea to have this feature be triggered by the `linkValidation` option.  Could you update the documentation to mention this additional functionality when `linkValidation` is used?  Check `README.md` and `OPTIONS.md` for references.